packages/hxd/hxd.0.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+name:         "hxd"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/hxd"
+bug-reports:  "https://github.com/dinosaure/hxd/issues"
+dev-repo:     "git+https://github.com/dinosaure/hxd.git"
+doc:          "https://dinosaure.github.io/hxd/"
+license:      "MIT"
+synopsis:     "Hexdump in OCaml"
+description: """Please, help me to debug ocaml-git
```

Comment:
I find this description funny :) but do you want to update it with something more descriptive before I merge, or it's fine?

Reply:
It's fine :+1:, the synopsis is clear and then, the goal of this package is very limited (debugging only, and used only by me).